modules/ROOT/pages/kafka/kafka-connector-reference.adoc

```diff
@@ -28,99 +33,100 @@ endif::[]
+| Key Store Type a| String |  The file format of the key store file. This is optional and default value is "JKS". |  JKS | 
+| Key Store Password a| String |  The store password for the key store file. This is optional and only needed if "keyStoreLocation" is configured. |  | 
+| Key Store Location a| String |  The location of the key store file. This is optional and can be used for two-way authentication for connector. |  | 
+| Trust Store Type a| String |  The file format of the trust store file. |  JKS | 
+| Trust Store Password a| String |  The password for the trust store file. |  | x
 | Trust Store Location a| String |  The location of the trust store file. |  | x
 |===
 
 
 ==== Associated Sources
-
-* <<consumer>>
+* <<consumer>> 
 
 ---
 [[kafka-producer-config]]
 === Producer configuration
 
 
 ==== Parameters
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
 |Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
-| Connection a| * <<kafka-producer-config_basic-kafka-producer-connection, Kafka Basic Producer Connection>>
-* <<kafka-producer-config_kerberos-kafka-producer-connection, Kafka Kerberos Producer Connection>>
-* <<kafka-producer-config_kerberos-ssl-kafka-producer-connection, Kafka Kerberos SSL Producer Connection>>
-* <<kafka-producer-config_ssl-kafka-producer-connection, Kafka SSL Producer Connection>>
- | The connection types that can be provided to this configuration. | | x
-| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform will expire the instance at the exact moment that it becomes eligible. The runtime will actually purge the instances when it sees it fit. |  |
+| Connection a| * <<kafka-producer-config_basic-kafka-producer-connection, Kafka Basic Producer Connection>> 
+* <<kafka-producer-config_kerberos-kafka-producer-connection, Kafka Kerberos Producer Connection>> 
+* <<kafka-producer-config_kerberos-ssl-kafka-producer-connection, Kafka Kerberos SSL Producer Connection>> 
+* <<kafka-producer-config_ssl-kafka-producer-connection, Kafka SSL Producer Connection>> 
+ | The connection types to provide to this configuration. | | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances when it sees fit. |  | 
```

Comment:
The problem sentence again...
The runtime purges the instances when it sees fit.

I wonder what basis the runtime has for "seeing fit" to purge an instance? If we knew that, it would be easier to reword it. Mule runtime purges the instance the next time this thing happens.